tests/tx_validation_test.go

```diff
@@ -663,7 +663,7 @@ func TestValidationInvalidSig(t *testing.T) {
 
 				// For block tx validation test
 				if expectedErr == blockchain.ErrInvalidFeePayer {
-					expectedErr = types.ErrInvalidSigFeePayer
+					expectedErr = types.ErrFeePayer(types.ErrInvalidSigFeePayer)
 				}
```

Comment:
This if statement is unnecessary.

Reply:
Thanks

Reply:
@blukat29 I have checked you need the condition because the expectedErr also can be for sender at
```go
tx, expectedErr := invalidCase.fn(t, txType, reservoir, contract, signer)
```